pkg/bugs/bugs.go

```diff
@@ -74,9 +74,6 @@ func (b Bug) BlockerRequested() bool {
 }
 
 func (b Bug) Untriaged() bool {
-	if b.Status == "NEW" {
-		return true
-	}
```

Comment:
I'm pretty sure @eparis mean to add this in #13, although I agree that it seems like a bug should be able to be triaged but not yet assigned to a particular dev.

Reply:
It definitely was on purpose, but lala, bparees, like, and a few others didn't like it. The point was to make sure we quickly put eyes on bugs opened medium or lower sev and priority. To make sure we agreed with that assessment.

Some teams apparently, I did not know this, use ASSIGNED to mean "I'm actively working on it" rather than how I've always thought we all used to, to mean "it's been made person X's problem"

I guess I'm ok trying this change and hoping the ebes review will eventually get all bugs the review they deserve.